include/abb_librws/rws_interface.h

```diff
@@ -204,17 +260,24 @@ class RWSInterface
   /**
    * \brief A method for collecting runtime information of the robot controller.
    *
-   * \return RuntimeInfo container for the runtime information.
+   * \return RuntimeInfo containing the runtime information.
```

Comment:
Hm. Perhaps personal, but I'd prefer the "Container for .." version here, as it describes *what* `RuntimeInfo` *is*, not what it does.

Reply:
That's resonable, I will think a bit about it.

Reply:
I will consider this in a future PR.